etc/ReplayOfflineConfiguration.py

```diff
@@ -628,6 +630,19 @@
                sizePerEvent=2250, # copied from JetHT - should be checked
                scenario=ppScenario)
 
+DATASETS = ["JetMET0", "JetMET1"]
```

Comment:
This is just cosmetic, but I would prefer to have 'JetMET', 'JetMET0', and 'JetMET1'. I would guess that, if `JetMET` were to be merged again, we would use the same config as for `JetMET0` and `JetMET1`

Reply:
Okay fixing!

Reply:
Done